src/Compilers/Core/CommandLine/CompilerServerLogger.cs

```diff
@@ -46,7 +46,7 @@ static CompilerServerLogger()
                     // Otherwise, assume that the environment variable specifies the name of the log file.
                     if (Directory.Exists(loggingFileName))
                     {
-                        loggingFileName = Path.Combine(loggingFileName, $"server.{loggingFileName}.{GetCurrentProcessId()}.log");
+                        loggingFileName = Path.Combine(loggingFileName, $"server.{GetCurrentProcessId()}.log");
```

Comment:
Rather than removing the name I think we need to change the code to distinguish between the cases when it's a file name or full path. In the case of the full path we could just use `Path.GetFileName` to trim this down to a name here and keep the current code valid.

Reply:
Eh, I don't see a lot of use in that. What information does the name give? PID seems more useful.